packages/styles/src/components/tag.scss

```diff
@@ -0,0 +1,53 @@
+@forward './../variables/options';
+@forward './utilities';
+
+@use './../variables/components/tag' as tag;
+
+.tag:has(:not(pre)), // is only a temporary fix till I know how to do it wihtout causing future issues. wihtout this the Code sample would break currently.
```

Comment:
The `:has(:not(pre))` selector fixes a problem that only exists in Storybook, therefore it should not be in the styles package.

It would be better to override the `.tag` styles in preview.scss.

Reply:
Yes, but...
The same problem could happen on project using code highlighting tools as well.